src/Pages/ErrorPage/style.module.css

```diff
@@ -0,0 +1,74 @@
+@import "../../colors.module.css";
+
+body{
+  margin: 0px;
+  padding: 0rem !important;
+}
+
+.body{
+  padding: 100px 0px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.rightBody{
+  display: flex;
+  align-items: end;
+}
+
+.buttonContainer{
+  display: flex;
+  justify-content: left;
+}
+
+h1, p {
+  color: var(--NAVY_BLUE);
+}
+
+h1 {
+  font: "open sans";
+  font-size: 10rem;
+}
+
+p{
+  font-size: 2rem;
+}
+
+.goBackButton{
+  width: 200px;
+  height: 77px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--BANANA_YELLOW);
+  color: var(--NAVY_BLUE);
+  font-family: 'Open Sans', sans-serif;
+  font-weight: bold;
+  font-size: 1.5rem;
+  border-radius: 8px;
+  border-color: var(--BANANA_YELLOW);
+  border-width: 0px;
+  margin: 10px 10px 10px 0px;
+}
+
+.homeButton{
+  width: 200px;
```

Comment:
anytime you style a `width`, `height`, `padding` or `margin` you need to be using either `rems` or `percentages`
pixels are very rigid and inflexible and this can lead to text overrunning the borders of its parent container or other weird styling issues on window resize

Reply:
https://www.reddit.com/r/web_design/comments/7t72wv/comment/dtadjwt/?utm_source=share&utm_medium=web2x&context=3